packages/core/src/components/Button/Button.scss

```diff
@@ -17,12 +17,15 @@
   text-decoration: none;
 
   &:focus,
-  &:hover {
+  &:hover,
+  &.ds-c-button--focus,
```

Comment:
Added 3 new button classes used in the code examples. `ds-c-button--focus`, `ds-c-button--hover`, and `ds-c-button--active`. 
`ds-c-button--focus` isnt used right now but once we implement focus we can add it to the code examples

Reply:
It also might also be worth refactoring `ds-c-button--disabled` to work similarly to those new classes. Could be included in this ticket: https://jira.cms.gov/browse/WNMGDS-290